needle/core/utils/constants.py

```diff
@@ -97,10 +97,26 @@ class Constants(object):
     # DEVICE
     # ==================================================================================================================
     # DEVICE PATHS
+<<<<<<< HEAD
+    DEVICE_PATH_TEMP_FOLDER      = '/var/root/needle/'
+    DEVICE_PATH_APPLIST_iOS7     = '/var/mobile/Library/Caches/com.apple.mobile.installation.plist'
+    DEVICE_PATH_APPLIST_iOS8     = '/var/mobile/Library/MobileInstallation/LastLaunchServicesMap.plist'
+    DEVICE_PATH_APPLIST_iOS9     = '/private/var/installd/Library/MobileInstallation/LastLaunchServicesMap.plist'
+    DEVICE_PATH_BUNDLE_iOS7      = '/private/var/mobile/Applications/'
+    DEVICE_PATH_BUNDLE_iOS8      = '/private/var/mobile/Containers/Bundle/Application/'
+    DEVICE_PATH_BUNDLE_iOS9      = '/private/var/containers/Bundle/Application/'
+    DEVICE_PATH_DATA_iOS8        = '/private/var/mobile/Containers/Data/Application/'
+    DEVICE_PATH_DATA_iOS9        = '/private/var/mobile/Containers/Data/Application/'
+    DEVICE_PATH_TRUST_STORE      = '/private/var/Keychains/TrustStore.sqlite3'
+    DEVICE_PATH_FRIDA_CACHE      = '/Library/Caches/frida-*'
+    DEVICE_PATH_HOSTS            = '/etc/hosts'
+    DEVICE_PATH_EFFECTIVE_CONFIG = '/var/mobile/Library/ConfigurationProfiles/EffectiveUserSettings.plist'

```

Comment:
Careful that this path is not the same on all the versions of iOS.
You may want to provide different alternatives based on the OS level.